frontend/src/component/common/UserAvatar/UserAvatar.tsx

```diff
@@ -77,7 +77,7 @@ export const UserAvatar: FC<IUserAvatarProps> = ({
             sx={sx}
             {...props}
             data-loading
-            alt='Gravatar'
+            alt={user?.name || user?.email || user?.username || 'Gravatar'}
```

Comment:
The previous alt wasn't very descriptive at all, so we'll add the user's name if it's present.

Reply:
I wish we could do it only in the backend so that the lookup logic doesn't leak out to frontend. I already collapse all of those to name for the created by field in search results. But I assume other places still expose the whole user that you need to dissect.

Reply:
Yeah, so this is in the existing user avatar component, which already uses this kind of messy logic everywhere. I don't really want to go look at all the places we use that component and what kind of data we might have gotten from the back end there. So this is just updating that component to show a slightly better alt text based on whatever data you feed it.

So ... yeah.